test/units/modules/network/asa/test_asa_og.py

```diff
@@ -0,0 +1,117 @@
+# This file is part of Ansible
```

Comment:
Please use the updated license as added [here](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/cli/cli_config.py#L1-L5), changing the year.